starwars/src/components/StarWars.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './StarWars.css';
+import StarWarsJSON from './StarWarsJSON.json';
+
+
+class StarWars extends React.Component {
+    
+    render() {
+        
+        return (
+            <ol>
+            {StarWarsJSON.map( character => {
+            return <li>{character.name}</li>
```

Comment:
While this technically hits the mvp requirements outlined in the readme, I would go a step further and render a new component for each character. I would also try displaying a few more properties for each character.

Reply:
I've gone ahead and restyled the css and added more data, as well creating a subcomponent for each character.  Thanks!